pages/create-account.js

```diff
@@ -10,17 +12,19 @@ export default function CreateAccount() {
     lastName: '',
     password: '',
     username: '',
+    email: ''
```

Comment:
Nit - Jack got me for the same thing last week, but lets move the email variable to after confirmPassword. Just to maintain alphabetic consistency.